Building.md

```diff
@@ -0,0 +1,43 @@
+This is the general getting started guide for FullScreenShenanigans.
+
+### Table of Contents
+
+1. [Obtaining the Source Code](#obtaining-the-source-code)
+2. [Build Process](#build-process)
+3. [Using the Console](#using-the-console)
+
+## Obtaining the Source Code
+
+In order to play FullScreenShenanigans, you first need to obtain the source code. There are two ways to do this:
+
+A. Clone the repository:
+1. Copy the clone URL (URL located to the left of the Download Zip button).
+2. Open Git Bash and change the current working directory to the location where you want the cloned directory to be made. 
+3. Type git clone and paste the URL you copied then press enter. You now have your own copy of FullScreenShenanigans! See [Build Process](#build-process) to see how to finish setup and play the game. 
+
+B. Download the latest release .zip of this project, extract that onto your computer, and open index.html in a browser (preferably Google Chrome). That's it!
+
+
+## Build Process
+
+FullScreenShenanigans uses [Grunt](http://gruntjs.com/) to automate building, which requires [Node.js](http://node.js.org). 
+
+Go to the root repository folder (containing the Getting Started guide) and run the following commands:
+
+    npm install
+    grunt
+    
+You may also have to manually install grunt first:
+
+    npm install -g grunt grunt-cli
+
+All of the files in Source, will be compiled into a new Distribution folder in the root.
+
+* FullScreenShenanigans.d.ts and FullScreenShenanigans.ts are merged into a FullScreenShenanigans.ts, and all .ts files under References are copied as well. A .js file with a .min.js and .min.js.map will be generated as well.
+* Distribution/FullScreenShenanigans-0.10.X is a minified version for use in static websites. It's also zipped for your convenience.
+
+You can now play the game by opening index.html in a browser (preferably Google Chrome)!
+
+
+## Using the Console
+You can use the console to customize your gaming experience. Simply type in FSS.some_function in order to make a change to the game. You have access to all Functions found in the FullScreenShenanigans.js file and can therefore add and remove Things, change around the maps and more!
```

Comment:
If you give someone something that looks like code they're going to try to run it, and then get confused because `FSS.some_function` isn't anything.

You can use FSM as an example, and show some generic commands like play/pause.